android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/requests/ShowConstantTbtTests.java

```diff
@@ -30,7 +30,7 @@
 
 /**
  * This is a unit test class for the SmartDeviceLink library project class :
- * {@link com.smartdevicelink.proxy.rpc.ShowConstantTbt}
+ * {@link ShowConstantTbt}
```

Comment:
@kostyaBoss On line 33 I would remove it and change it back to `{@link com.smartdevicelink.proxy.rpc.ShowConstantTbt}`

The idea is that nothing should be changed in this class.